src/Oscoin/Node/Options.hs

```diff
@@ -81,7 +81,7 @@ nodeOptionsParser cps = Options
                \the development environment, and is useful to avoid busy looping \
                \in an idle network."
        <> metavar "SECONDS"
-       <> value 1
+       <> value (1 * seconds)
```

Comment:
This now shows `(default: 1000000000)` in the help text. I guess we need to revert this and `map (* seconds)` over the whole option.

Reply:
Yeah, our Go `Duration` is just so utterly useless. I'll change this to a `Word8`.

Reply:
_Alfredo casually drops https://github.com/oscoin/oscoin/issues/518 and leaves. 😬_